.eslintignore

```diff
@@ -0,0 +1,11 @@
+node_modules/
```

Comment:
Thoughts on alternatives to this file?

Today, eslint uses `.gitignore` as its [`ignore-path` parameter](https://github.com/fusionjs/fusion-plugin-jwt/blob/master/package.json#L52).  What we really want is a superset that includes `/flow-typed/npm/**` as we want it ignored by eslint but not by git.

Reply:
I'd even vote to leave it as-is and just add eslint-ignore headers to the top of flow-typed files. Bit of a manual step, but keeps the eslint config consistent across all of our repos.

Reply:
Yeah, I thought about that as well.  I'm generally not a fan of modifying what amounts to "external repository" code (_node_modules_, _flow-typed_, ...) but it may be preferable to maintaining both of these ignore files.